### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by varsill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,21 @@ To update your existing installation, invoke `make install` again.
...
+It's advisable to use the `--workspace` and `--profile` options when running Sebex or to set the corresponding env vars: `SEBEX_WORKSPACE` and `SEBEX_PROFILE`.
++ The workspace is a directory, where sebex clones repositories, so that it can work on them later on. It also contains `.sebex` subdirectory, with some metadata files needed for sebex to run properly, and the `profiles` subdirectory, where the list of profiles specification is put. The default workspace (appicable to a situation that you haven't explicitly specify the workspace) is the directory, from which you are running the sebex.
++ The profile helps you manage different sebex configurations. Each profile is described by the `<profile name>` text file inside the `<workspace_directory>/profiles/` directory. You need to create that file on your own. An exemplary profile description file can be found inside `examples/` subdirectory of this repository. Note, that the `<profile name>` is exactly the same as the profile name specified by the `--profile` option - that means, that the file does not have any special extension.
```
Comment (mat-hek) [inline README.md:29]: fix the part about extension

File: README.md [hunk 2]
```diff
@@ -57,8 +60,8 @@ Project: sebex_test_e
 Project:
```
Comment (mat-hek) [inline README.md:60]: also versions are passed here

File: README.md [hunk 3]
```diff
@@ -80,14 +83,18 @@ Release "Purely Easy Wahoo"
 
 Save this release? [y/N]:
 ```
+The release plan will be saved inside the `<workspace director>/release.yaml` file.
```
Comment (mat-hek) [inline README.md:86]:
```suggestion
The release plan will be saved inside the `<workspace directory>/release.yaml` file.
```

**Review by mat-hek — CHANGES_REQUESTED** [review]
Please run grammarly on that

File: README.md (hunk 1 above)
Comment (varsill) [reply README.md:29]: done

File: README.md (hunk 3 above)
Comment (varsill) [reply README.md:86]: done.
I have also changed the directory path to `<workspace directory/.sebex/release.yaml`

**Pull request merged**